crates/router/src/core/refunds/validator.rs

```diff
@@ -125,7 +126,7 @@ pub async fn validate_uniqueness_of_refund_id_against_merchant_id(
 pub fn validate_refund_list(limit: Option<i64>) -> CustomResult<i64, errors::ApiErrorResponse> {
```

Comment:
You can accept an `offset` parameter here itself instead of using another validator function?